packages/relay/src/lib/clients/clientCache.ts

```diff
@@ -101,6 +101,12 @@ export class ClientCache {
         this.cacheKeyGauge.labels('', ClientCache.setCacheLabel, callingMethod || '').inc(1);
     }
 
+    public delete(key: string, callingMethod: string, requestIdPrefix?: string): void {
+        this.logger.trace(`${requestIdPrefix} delete cache for ${key}`);
+        this.cache.delete(key);
+        this.cacheKeyGauge.labels('', ClientCache.setCacheLabel, callingMethod || '').inc(1);
```

Comment:
This isn't a set. You can add a new deleteCacheLabel or better yet remove the metric increment